<?xml version="1.0"?>
<launch>
    <arg name="uart_id" default="/dev/ttyUSB1" /> 
    <arg name="sample_freq" default="60" />
    <arg name="buffer_time" default="0.25" /> 
    <arg name="publish_freq" default="60" /> 
    <arg name="node_name" default="imu_streamer" />
    <!-- <node pkg="multicam_server" type="imu_streamer.py" name="$(arg node_name)" output="screen"> -->
    <node pkg="multicam_server" type="imu_socket_server.py" name="$(arg node_name)" output="screen">
        <param name="publish_freq" type="double" value = "$(arg publish_freq)" /> 
        <param name="uart_id" type="string" value="$(arg uart_id)" /> 
        <param name="sample_freq" type="int" value="$(arg sample_freq)" /> 
        <param name="buffer_time" type="double" value="$(arg buffer_time)" /> 
    </node>
</launch>
